lib/project_types/script/config/extension_points.yml

```diff
@@ -35,12 +35,16 @@ tax_filter:
 payment_methods:
   domain: 'checkout'
   assemblyscript:
-    repo: "https://github.com/Shopify/extension-points.git"
+    repo: "https://github.com/Shopify/scripts-apis-examples"
     package: "@shopify/scripts-checkout-apis"
     toolchain-version: "^5.0.0"
+  javascript:
+    repo: "https://github.com/Shopify/scripts-apis-examples"
```

Comment:
should we add `beta: true` here so that it isn't exposed publicly for now?

Reply:
They would have to run `shopify config feature scripts_beta_languages --enable` in order to make it available

Reply:
Added the beta flag